pkg/controller/issueapi/logic.go

```diff
@@ -82,24 +82,24 @@ func (c *Controller) issue(ctx context.Context, authApp *database.AuthorizedApp,
 		}, nil
 	}
 
-	// Validate that the request with the provided test type is valid for this realm.
-	if !realm.ValidTestType(request.TestType) {
+	// Verify the test type
+	request.TestType = strings.ToLower(request.TestType)
+	if _, ok := c.validTestType[request.TestType]; !ok {
 		return &issueResult{
 			obsBlame:    observability.BlameClient,
-			obsResult:   observability.ResultError("UNSUPPORTED_TEST_TYPE"),
+			obsResult:   observability.ResultError("INVALID_TEST_TYPE"),
 			httpCode:    http.StatusBadRequest,
-			errorReturn: api.Errorf("unsupported test type: %v", request.TestType).WithCode(api.ErrUnsupportedTestType),
+			errorReturn: api.Errorf("invalid test type").WithCode(api.ErrInvalidTestType),
```

Comment:
Hmmm - this feels like an API change. Is that intended?

Reply:
Yea I think is what was intended / documented

Reply:
Ack - let's call this out in the changelog. I have paranoia about people depending on broken behaviors given this project's history.